transform/__tests__/i90_market_specific_transform.py

```
@@ -0,0 +1,46 @@
+from transform.i90_transform import TransformadorI90
+import time
+
+def test_market_specific_intradiario():
+    """
+    Test the transformation of a single day (latest day) for intradiario market
+    """
+    transformer = TransformadorI90()
+    result = transformer.transform_data_for_all_markets(dataset_type="volumenes_i90", mercados_lst=["intra"])
+    result_status = result["status"]
+
+    assert result_status["success"] == True, f"Market specific intradiario transformation failed. Details: {result_status.get('details', {})}"
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if result_status["success"]:` for truth checks

Replace with `result_status["success"]`

(E712)